### ISSUE TITLE
how to interpret the result of cocoVal2017 is better your repo?

### ISSUE THREAD

**Issue by Will-Hui** [Microsoft/human-pose-estimation.pytorch#79]
firstly, i will appreciate this repo.
when i run the  

    `python pose_estimation/valid.py\
     --cfg experiments/coco/resnet101/384x288_d256x3_adam_lr1e-3.yaml \
     --flip-test \
     --model-file models/pytorch/pose_coco/pose_resnet_101_384x288.pth.tar`

i can get the result as  fallows
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.755
 Average Precision  (AP) @[ IoU=0.50         | area=   all | maxDets= 20 ] = 0.925
 Average Precision  (AP) @[ IoU=0.75         | area=   all | maxDets= 20 ] = 0.826
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.724
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.808
 Average Recall       (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.784
 Average Recall       (AR) @[ IoU=0.50         | area=   all | maxDets= 20 ] = 0.936
 Average Recall       (AR) @[ IoU=0.75         | area=   all | maxDets= 20 ] = 0.845
 Average Recall       (AR) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.749
 Average Recall       (AR) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.838

this result is better your repo, my  configuration include : Python3.6 pytorch0.4.0
How to interpret this phenomenon? I do not modify anything about code, However, My valid result is better your repo result and the differenc is up to 1.9%

**Comment by leoxiaobin** [comment]
Please set TEST.USE_GT_BBOX to False. Your results use the ground truth bbox.

**Comment by Will-Hui** [comment]
> Please set TEST.USE_GT_BBOX to False. By default, I set TEST.USE_GT_BBOX to True. Your results use the ground truth bbox. That's why your results are better than which I reported.

thanks so much. and could you tell me what is the human detector in the first stage? 
Whether there is the use of external data in the training? for example, AIchallenger keypoints datasets

**Comment by leoxiaobin** [comment]
The human detector is trained on COCO dataset, using a Mask-RCNN architecture.

No external data is used for pose estimation.
